packages/govuk-frontend/src/govuk/components/header/header.mjs

```diff
@@ -128,10 +138,18 @@ export class Header extends Component {
     if (this.mql.matches) {
       this.$menu.removeAttribute('hidden')
       this.$menuButton.setAttribute('hidden', '')
+      if (this.$logo) {
```

Comment:
Rather than having another thing to keep in sync, could we just add an 'up to desktop' media query to the CSS, then add the class in the constructor after we return if there's no menu button?

Reply:
If we went with this approach I'd suggest setting the modifier on the `govuk-header` element and calling it something like `govuk-header--with-js-navigation`.

Reply:
I've updated with this approach - much cleaner.